### DISCUSSION TITLE
Icons for Microsoft Word, Excel, Powerpoint, and Teams

### DISCUSSION THREAD

**Discussion by Carssons** [elrumo/macOS_Big_Sur_icons_replacements#1326] [Icon request] [answered]
Greetings everyone,

These icons look really nice (please see attached). Please could you do me a favour by redesigning them in Big Sur style? 

I cannot find a similar icon for Microsoft Teams. I would be greatly grateful if someone could make a Teams icon in a similar style.

Thanks so much!

Regards.

![Office-2021](https://user-images.githubusercontent.com/86297154/142143279-f0573c69-2fa7-44b8-9049-c2d9ad2f5477.jpeg)

**Comment by KElee01** [comment, marked as answer]
Just made the Word icon, how do you like it?
![Word Icon](https://i.loli.net/2021/11/23/iusQnjaDMYRkm6B.png)

**Reply by Carssons** [reply]
Wow, it looks gorgeous. Thank you for your time and effort. I hope you have submitted it for approval.

It would be great if you could design another version with a white background, alongside with this version, for people to choose from. 

I look forward to your other icons.
Thanks again.
Cheers.

**Reply by KElee01** [reply]
Sure, I'll make the version with a white background later.
I just finished the other four icons and added a little detail to them, I hope you like them, or I can make them as similar to the original ones as I can.
Here are the icons:
![Word](https://i.loli.net/2021/11/23/q91tUKNVYS6rsM4.png)
![PowerPoint](https://user-images.githubusercontent.com/54350573/143011107-9cfc74ee-b654-4810-8cdf-b79a0cf50157.png)
![Excel](https://i.loli.net/2021/11/23/lKXxB1fCwLcvHZM.png)
![OneNote](https://i.loli.net/2021/11/23/W4xVQe8Ti5jsPAr.png)
![Teams](https://i.loli.net/2021/11/23/kmoRvIpV5ycYtwn.png)

**Reply by Carssons** [reply]
I love the icons, bro. You are so helpful. Thank you so much. You have designed a Teams icon, which I really appreciate.

Hope I can use these beautiful icons soon.

**Reply by KElee01** [reply]
I'm glad you like it. This is the version with white background:
![Microsoft Office Word_alt](https://user-images.githubusercontent.com/54350573/143009443-7e90af7a-7713-4898-9fcf-0a34cfb427a7.png)
![Microsoft Office OneNote_alt](https://user-images.githubusercontent.com/54350573/143009452-a0faf4e8-cbfe-45b7-b143-9a98abcacee7.png)
![Microsoft Office Teams_alt](https://user-images.githubusercontent.com/54350573/143009454-de55ccfb-a363-4b29-8a2e-e21d7f79db83.png)
![Microsoft Office PowerPoint_alt](https://user-images.githubusercontent.com/54350573/143009455-1e9dc22b-c157-4cfb-83cd-7bce02393a1f.png)
![Microsoft Office Excel_alt](https://user-images.githubusercontent.com/54350573/143009460-309ba7ea-b7dc-4c37-b419-31dd1a52ef62.png)
BTW, I just found there's somehow a colored line in the first version of the PowerPoint icon, I've already replace the icon above.

**Comment by Carssons** [comment]
These icons look great! You are excellent, bro. Thank you so much!

**Comment by bsilvamm** [comment]
Hi. Indeed these icons are fantastic. Can you make also for outlook.

**Reply by KElee01** [reply]
![Outlook](https://user-images.githubusercontent.com/54350573/143793333-c5749786-187c-4449-8fd1-6fdd223d65b5.png)
![Outlook_alt](https://user-images.githubusercontent.com/54350573/143793342-2495b90b-52df-4bdb-89ce-a2f8d8d23f9e.png)
Sorry, just spent some days figuring dealing with the details.

**Reply by bsilvamm** [reply]
Thanks. Awesome work friend! I hope you have submitted it for approval also.

**Reply by KElee01** [reply]
Yeah and it's already been approved. You can see all these icons at [HERE](https://macosicons.com/MicrosoftOffice), my username is `Kama`.

**Comment by KElee01** [comment]
I just remade the white background version again, they have colored shadow now, I hope you guys like them!
![Microsoft Office Word_alt](https://user-images.githubusercontent.com/54350573/144012800-f6b06a7f-acc8-4490-a28d-98eb457a5492.png)
![Microsoft Office OneNote_alt](https://user-images.githubusercontent.com/54350573/144012806-ed0d1227-2408-4837-83f1-5384b96c1753.png)
![Microsoft Office Outlook_alt](https://user-images.githubusercontent.com/54350573/144012810-162765ef-f695-43b3-b3be-79856745a060.png)
![Microsoft Office Teams_alt](https://user-images.githubusercontent.com/54350573/144012815-e5365199-595d-403e-bd44-f82405ac8ec0.png)
![Microsoft Office PowerPoint_alt](https://user-images.githubusercontent.com/54350573/144012822-6655c1ae-8f5c-437f-98e6-7d6901e7f4f8.png)
![Microsoft Office Excel_alt](https://user-images.githubusercontent.com/54350573/144012827-5ab54896-5375-460b-95e0-5804a206dec4.png)

**Reply by bsilvamm** [reply]
Yes awesome. Nice re-touch friend. Very creative.

**Reply by Carssons** [reply]
They look great, bro. I love them.

**Comment by Justonly1564** [comment]
Hello your icons are so gorgeous (I'm currently use them) don't you mind if you can do a similar icon but for one drive or adobe acrobat?

**Reply by KElee01** [reply]
Thank you, I've started on the onedrive icon already, but the acrobat icon is quite troublesome to make since the official logo (the curve thing) is a abstracted letter A, I can't put another "A" with it. So I probably can't make a similar icon for adobe acrobat. However, you can still download the acrobat icon I've already made at [here](https://github.com/KElee01/macos-icons/blob/main/Adobe%20Acrobat.icns) if you like it.

**Reply by KElee01** [reply]
The OneDrive icons are done!
![OneDrive_alt](https://user-images.githubusercontent.com/54350573/172633911-91198ec4-5565-4c13-94b4-ec70e0186345.png)
![OneDrive](https://user-images.githubusercontent.com/54350573/172633945-81479b90-1806-4310-8bc9-1d4ca5a4b284.png)
Star [my repo](https://github.com/KElee01/macos-icons) if you like them, please.

**Reply by Justonly1564** [reply]
WAOUH thanks you a lot your icon are just incredible.
Btw i see some icon in this style![Capture d’écran 2022-06-13 à 19.43.26.png](https://user-images.githubusercontent.com/100730110/173414711-d9b7b4bd-ebbf-45d0-bfde-0820273aabf1.png)
![Capture d’écran 2022-06-13 à 19.44.33.png](https://user-images.githubusercontent.com/100730110/173414752-1e5dee3e-44f6-4282-adf1-8f3ffa736671.png)
Is it possible to make icons in the same style for PowerPoint, OneNote, Outlook and OneDrive?

**Reply by KElee01** [reply]
Sorry, I don't think I can make this kind of skeuomorphism icon very well for now. It's pretty hard to make it looks real.
